src/svg/rich/cam-group-small.svg

```diff
@@ -0,0 +1,731 @@
+<svg width="97" height="90" viewBox="0 0 97 90" xmlns="http://www.w3.org/2000/svg">
+<defs>
+<style>
+.icon-blink {
+  animation: blink-animation 1s steps(5, start) infinite;
+  -webkit-animation: blink-animation 1s steps(5, start) infinite;
+}
+
+.icon-blink2 {
+  animation: blink-animation 1s steps(5, start) infinite;
+  -webkit-animation: blink-animation 1s steps(5, start) infinite;
+  animation-delay: 500ms;
+}
+
+.icon-blink3 {
+  animation: blink-animation 1s steps(5, start) infinite;
+  -webkit-animation: blink-animation 1s steps(5, start) infinite;
+  animation-delay: 350ms;
+}
+
+.icon-blink4 {
+  animation: blink-animation 1s steps(5, start) infinite;
+  -webkit-animation: blink-animation 1s steps(5, start) infinite;
+  animation-delay: 450ms;
+}
+
+.icon-blink5 {
+  animation: blink-animation 1s steps(5, start) infinite;
+  -webkit-animation: blink-animation 1s steps(5, start) infinite;
+  animation-delay: 500ms;
+}
+
+#camara-numer {
+  stroke-dasharray: 5;
+  stroke-dashoffset: 2;
+  animation: dash 2s linear forwards;
+  -webkit-animation: dash 2s linear forwards;
+}
+
+.test-stroke {
+  fill: none;
+  stroke: #000;
+  stroke-width: 2;
+  stroke-miterlimit: 5;
+}
+
+.scale-height {
+  animation-duration: 3000ms;
+  animation-iteration-count: infinite;
+  transform-origin: 50% 86%;
+  animation-name: scale-height;
+}
+
+/* .scale-height1 {
```

Comment:
Guess we need a task for cleaning up in the svg codes

Reply:
We have svg optimizer which cleanup and minify code automatically.